db/models.py

```diff
@@ -14,3 +14,37 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="acted_in_movies")
+    genres = models.ManyToManyField(Genre, related_name="movies_with_genre")
```

Comment:
it is better to use class name in plural here
how do u think what is better ? 
`genre.movies.all` or `genre.movies_with_genre.all`